nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-nar-utils/src/main/java/org/apache/nifi/nar/InstanceClassLoader.java

```diff
@@ -88,7 +89,7 @@ public synchronized void setInstanceResources(final URL[] urls) {
         if (shimClassLoader != null) {
             try {
                 c = shimClassLoader.loadClass(name, resolve);
-            } catch (ClassNotFoundException cnf) {
+            } catch (NoClassDefFoundError | ClassNotFoundException e) {
```

Comment:
Good catch, I think I added that when I was troubleshooting something and forgot to remove it, I can take that out